block/manager.go

```diff
@@ -25,79 +22,55 @@ import (
 
 	"github.com/dymensionxyz/dymint/config"
 	"github.com/dymensionxyz/dymint/da"
-	"github.com/dymensionxyz/dymint/log"
 	"github.com/dymensionxyz/dymint/mempool"
 	"github.com/dymensionxyz/dymint/settlement"
 	"github.com/dymensionxyz/dymint/state"
 	"github.com/dymensionxyz/dymint/store"
 	"github.com/dymensionxyz/dymint/types"
 )
 
-type blockSource string
-
-const (
-	producedBlock blockSource = "produced"
-	gossipedBlock blockSource = "gossip"
-	daBlock       blockSource = "da"
-)
-
-type blockMetaData struct {
-	source   blockSource
-	daHeight uint64
-}
-
 // Manager is responsible for aggregating transactions into blocks.
 type Manager struct {
```

Comment:
only change is`syncCache` removed as it's not used
grouping and comments other than that